src/core/tools/common/simple_perf_counter.cpp

```diff
@@ -271,9 +272,9 @@ namespace dsn {
                     _timer->expires_from_now(boost::posix_time::seconds(_counter_computation_interval_seconds));
                     _timer->async_wait(std::bind(&perf_counter_number_percentile::on_timer, this, std::placeholders::_1));
                 }
-                else
+                else if (boost::system::errc::operation_canceled != ec)
```

Comment:
fixed in https://github.com/qinzuoyan/rDSN/commit/1961ec28cab20607210a3afb26d927fb46a52b1a